File: doc-examples/nitrite-example-java/src/main/java/example/GeometryTypesExample.java
package example;

import org.locationtech.jts.geom.Coordinate;
import org.locationtech.jts.geom.Geometry;
import org.locationtech.jts.geom.GeometryFactory;
import org.locationtech.jts.geom.Point;

// tag::geometry-types[]
public class GeometryTypesExample {

    public static void main(String[] args) {
        GeometryFactory factory = new GeometryFactory();

        // Create a Point - a single coordinate (x, y)
        // IMPORTANT: JTS uses (longitude, latitude), NOT (latitude, longitude)
        Point nyc = factory.createPoint(new Coordinate(-74.0060, 40.7128));

        // Use the geometry in your entity
        // entity.setLocation(nyc);
    }
}
// end::geometry-types[]

File: doc-examples/nitrite-example-kotlin/src/main/kotlin/example/GeometryTypesExample.kt
package example

import org.locationtech.jts.geom.Coordinate
import org.locationtech.jts.geom.GeometryFactory
import org.locationtech.jts.geom.Point

// tag::geometry-types[]
object GeometryTypesExample {

    @JvmStatic
    fun main(args: Array<String>) {
        val factory = GeometryFactory()

        // Create a Point - a single coordinate (x, y)
        // IMPORTANT: JTS uses (longitude, latitude), NOT (latitude, longitude)
        val nyc: Point = factory.createPoint(Coordinate(-74.0060, 40.7128))

        // Use the geometry in your entity
        // entity.setLocation(nyc)
    }
}
// end::geometry-types[]
